### DISCUSSION TITLE
Where do these classes come from? "vjs-overlay-top-left vjs-overlay-no-background"

### DISCUSSION THREAD

**Discussion by tzwel** [FreeTubeApp/FreeTube#5399] [General]
I want to contribute, but I can't find either `vjs-overlay-top-left`  or `vjs-overlay-no-background` anywhere in the source code
Not in the player css/js, a github code search doesn't help either

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @tzwel just curious why are you looking for that? Is there a certain issue you like to contribute to?

**Reply by tzwel** [reply]
I have written a custom css that adds this youtube-like gradient to the top of the player in fullscreen so the text is much more legible and I want to implement this properly

![image](https://github.com/user-attachments/assets/8d1fe037-fc65-4ba6-ae05-9c46071a5cbf)

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Ah I see that would address  https://github.com/FreeTubeApp/FreeTube/issues/1348. 

In our next major release we will migrate our video player from video.js to shaka see https://github.com/FreeTubeApp/FreeTube/pull/4978 therefore we will not accept any PR's that are related to the player. The migration addresses the issue you are trying to solve.

We appreciate that you would like to contribute and hopefully this doesn't discourage you into making contributions in the future

**Reply by tzwel** [reply]
Good! I could have done more research about this. I certainly will contribute if I feel like FreeTube is missing something
@efb4f5ff-1298-471a-8973-3d47447115dc

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Anything specific you're thinking about. Maybe I can direct you to some issues based on your interests

**Reply by tzwel** [reply]
I don't know now. I'll let you know once I think of something. But you can point me to some style-related issues for a starter

**Reply by tzwel** [reply]
Thinking of implementing some kind of an audio compressor (akin to the new youtube's stable audio option) and other audio related stuff like outputting only the mono signal as I found in one of the issues

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
> I don't know now. I'll let you know once I think of something. But you can point me to some style-related issues for a starter

Sure :)

- https://github.com/FreeTubeApp/FreeTube/issues/5090
- https://github.com/FreeTubeApp/FreeTube/issues/5233
- https://github.com/FreeTubeApp/FreeTube/issues/5211
- https://github.com/FreeTubeApp/FreeTube/issues/5400
- https://github.com/FreeTubeApp/FreeTube/issues/4852
- https://github.com/FreeTubeApp/FreeTube/issues/4821
- https://github.com/FreeTubeApp/FreeTube/issues/4536
- https://github.com/FreeTubeApp/FreeTube/issues/4535
- https://github.com/FreeTubeApp/FreeTube/issues/4284
- https://github.com/FreeTubeApp/FreeTube/issues/4162
- https://github.com/FreeTubeApp/FreeTube/issues/4155
- https://github.com/FreeTubeApp/FreeTube/issues/3530
- https://github.com/FreeTubeApp/FreeTube/issues/3313
- https://github.com/FreeTubeApp/FreeTube/issues/3288
- https://github.com/FreeTubeApp/FreeTube/issues/3254
- https://github.com/FreeTubeApp/FreeTube/issues/3205
- https://github.com/FreeTubeApp/FreeTube/issues/3204
- https://github.com/FreeTubeApp/FreeTube/issues/2589
- https://github.com/FreeTubeApp/FreeTube/issues/2043
- https://github.com/FreeTubeApp/FreeTube/issues/1451
- https://github.com/FreeTubeApp/FreeTube/issues/1450
- https://github.com/FreeTubeApp/FreeTube/issues/1356
- https://github.com/FreeTubeApp/FreeTube/issues/1092
- https://github.com/FreeTubeApp/FreeTube/issues/693#issuecomment-1824975963
